test/index-test.js

```diff
@@ -1,18 +1,82 @@
 const expect = require('expect')
+const createSpy = expect.createSpy;
 
 const fs = require('fs')
 const jsdom = require('mocha-jsdom')
 const path = require('path')
 
 
 describe('index', () => {
-  
+
+  const sourceCode = fs.readFileSync(path.resolve(__dirname, '..', 'index.js'), 'utf-8');
+
   jsdom({
-    src: fs.readFileSync(path.resolve(__dirname, '..', 'index.js'), 'utf-8')
+    src: sourceCode,
+  })
+
+  describe('`countdown` function', function () {
+    it('should exist', function () {
+      expect(countdown).toExist()
+    })
+
+    it('should have call the given callback function after two seconds', function (done) {
+      this.timeout(3000);
+      const spy = createSpy()
+      countdown(spy)
+
+      expect(spy.calls.length).toEqual(0)
```

Comment:
Should we use `expect(spy).toNotHaveBeenCalled()` for consistency? (Bummer that expect doesn't have `toHaveBeenCalledOnce()` for below.)

(Also that the `not` methods use split infinitives... anyway.)